TASVideos/wwwroot/css/partials/_customizations.scss

```diff
@@ -83,6 +83,29 @@ article.wiki table {
 	}
 }
 
+/* Forum Tables */
+.postbody table {
+	width: 100%;
+	margin-bottom: 1rem;
+	color: var(--bs-dark);
+	border-collapse: collapse;
+}
+
+.postbody th, td {
+		padding: 0.3rem;
+		vertical-align: top;
+		border: 1px solid var(--bs-gray-600);
+}
+
+.postbody tbody th {
+		background-color: rgba(var(--bs-black-rgb), 0.05);
+}
+
+.postbody tbody + tbody {
+		border-top: 2px solid #dee2e6;
```

Comment:
`#dee2e6` should be either a SASS variable like `$dropdown-text-dark` if not intended to change in dark mode, or a native css var if it needs to change in dark mode (good use of those otherwise though)